services/client/src/pages/events/utils/index.js

```diff
@@ -8,3 +9,7 @@ export function updateEventsList(eventsList) {
 
   return _.orderBy(updatedEvents, ["timestamp"], ["asc"]);
 }
+
+export const parsedDomain = parseDomain(window.location.origin, {
+  customTlds: /localhost/
```

Comment:
From reading the `parse-domain` [docs](https://www.npmjs.com/package/parse-domain), we will need to make this configurable as the production muxer.co.uk domain will need to be specified here. The docs also have some useful examples of tests that would be good to add to ensure this functions as expected.

Reply:
I think that `customTlds` option is to allow custom domains like `localhost`, am I wrong?